CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 The GraphQL Accounts project was intended - since it's inception - to be a community maintained project.  We'd love to see you get involved (especially long time contributors from the Meteor community who we've worked with before).
```

Comment:
Do you mind rephrasing this so it doesn't include "GraphQL" since we're transport agnostic.